content/guides/outliers.md

```diff
@@ -37,11 +37,11 @@ To set up an outlier detection graph for your data you add a metric to the graph
 
 To do so, create a new timeseries graph on your dashboard with your chosen metric. Your screen should look like:
 
-![](/static/images/outliers/outliers-dash-choose-metrics-newer.png)
+![](/static/images/outliers/outliers-dash-choose-metrics-updated.png)
 
-Now, click on the + icon (Add functions and modifiers) on the right side of the second metrics line. In the "Modify your query" box, choose the "outliers" function:
+Now, click on the + icon (Add functions and modifiers) on the right side of the metrics line. In the popup box, choose "Algorithms" then one of the four outlier algorithms:
```

Comment:
Now, click on the + icon on the right side of the metrics line. Choose "Algorithms" from the function categories, then one of the four outlier algorithms: